gunrock/app/enactor_base.cuh

```diff
@@ -646,11 +646,14 @@ public:
             {
                 printf("Loading Matrix-market coordinate-formatted graph ...\n");
             }
+
             char *market_filename = args.GetCmdLineArgvDataset();
-            if (market_filename == NULL)
+
+            std::ifstream fp(market_filename);
+            if (market_filename == NULL||!fp.is_open())
             {
                 fprintf(stderr, "Input graph does not exist.\n");
```

Comment:
I agree with @jowens on adding the filename to stderr. Have you run ctest after the change? Also, please test for all primitives for a bogus filename or an empty filename, and see if the output make sense. (You can post the output here, you can and you should write a simple shell/python script for your test too)

As for pushing to the master, @wetliu I might have told you to push to the master, if so I apologize, according to https://github.com/gunrock/gunrock/issues/85 master is for actual release. So please cherrypick this commit to the dev branch of your fork and do the things I suggested, then submit the pull request again. Thank you!